src/Modules/OrchardCore.Commerce.Payment.Stripe/Assets/Scripts/stripe-payment-form.js

```diff
@@ -107,7 +104,7 @@ window.stripePaymentForm = function stripePaymentForm(
                         });
                 }
 
-                const validationJson = await fetchPost(validateUrl);
+                const validationJson = await fetchPost(validateUrl + '/' + paymentIntentId);
```

Comment:
```suggestion
                const validationJson = await fetchPost($"{validateUrl}/{paymentIntentId}");
```

Reply:
That's not valid Javascript. :melting_face: